### DISCUSSION TITLE
Make combobox scroll selected item into view when opening.

### DISCUSSION THREAD

**Discussion by alex-delia** [shadcn-ui/ui#5496] [Q&A]
Hey everyone! Title pretty much sums it up, is there a way to make the combobox scrollIntoView the selected value when opening it? I've wasted a few hours trying to figure this out through different combinations of attaching refs to the Command components and the command items themselves with no luck. Thanks!

**Comment by ariefsn** [comment]
This works for me

```
  useEffect(() => {
    if (open && selected) {
      setTimeout(() => {
        const els = document.getElementsByClassName('mySelectedClassName');
        if (els.length > 0) {
          els[0].scrollIntoView({ behavior: 'smooth' });
        }
      }, 100)
    }
  }, [open, selected])
```

And add the `mySelectedClassName` to `CommandItem`'s `className`
